content/services/services_powerdev.rst

```diff
@@ -40,4 +30,150 @@ quota, but can increase given resource availability and justification.
++-----------------------------+------------------------------------------------+
+|UMBC                         | Supporting University of Maryland, Baltimore   |
+|                             | County work on the POWER8 architecture.        |
++-----------------------------+------------------------------------------------+
+|Zarafa                       | Supporting porting efforts of POWER8           |
+|                             | ppc64/ppc64le for the Zarafa project.          |
++-----------------------------+------------------------------------------------+
+
+Academic Partners
+~~~~~~~~~~~~~~~~~
+
++-----------------------------+------------------------------------------------+
+|University of Alberta and    | The goal is to investigate the use of the      |
+|Universidade de Campinas     | speculation support in POWER8 for the speeding |
+|                             | up the sequential execution of programs.       |
+|                             | Single-threaded speculation has been used in   |
+|                             | the past, through trace-based compilation. The |
+|                             | goal of this project is to investigate         |
+|                             | multi-threaded speculation of alternative paths|
+|                             | of execution in a sequential execution.        |
+|                             |                                                |
+|                             | This is a joint project between University of  |
+|                             | Alberta (UofA) and Universidade de Campinas    |
+|                             | (Unicamp).                                     |
++-----------------------------+------------------------------------------------+
+|Berkeley Lab                 | BLCR is "Berkeley Lab Checkpoint/Restart", a   |
+|Checkpoint/Restart           | kernel-level checkpointer implemented via a    |
+|                             | loadable kernel module and a userspace library.|
+|                             |                                                |
+|                             | Project home: http://ftg.lbl.gov/checkpoint    |
++-----------------------------+------------------------------------------------+
+| Harvey Mudd College         | Observationally Cooperative Multithreading     |
+|                             | (OCM) provides a "kinder gentler" form of      |
+|                             | concurrency, allowing programmers to imagine   |
+|                             | that a single thread runs on the machine at any|
+|                             | one time. With POWER8 hardware transactional   |
+|                             | memory, it becomes possible to actually run    |
+|                             | multiple threads at the same time while        |
+|                             | *appearing* to be running only one at a time.  |
++-----------------------------+------------------------------------------------+
```

Comment:
Can we change this to using the ``csv-table`` type? You can use an external csv file to source. Doing it this way makes it difficult to manage in the long term. If you take a look at our internal docs repo, you can see a few use cases.